Codecademy/gamut
# fix(LayoutMenuVariant): change border color of accordion area

### Overview


Updates border color of the accordion area to navy-300 as per design request



### PR Checklist

- [X] Related to JIRA ticket: [DISC-1069]
NOTE: I'm seeing failures unrelated to my gamut changes in both Codecademy & Mono right now...

#### Testing Instructions



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

Slack conversation for reference: https://codecademy.slack.com/archives/G015E08LW3T/p1675693102223139
Screenshot since it's from a thread in a private channel:
<img width="732" alt="Screen Shot 2023-02-06 at 1 58 26 PM" src="https://user-images.githubusercontent.com/9753446/217060709-cd6298fe-8c7c-4b35-9550-59e26ea9a1c9.png">

1. Visit storybook and see the updated divider color in the LayoutMenuVariant component
2. Visit the mono preview environment and note the updated color
3. Approve!

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34578)  | [Monolith Env](https://pr-34578-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/1211)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1211)   |




[DISC-1069]: https://codecademy.atlassian.net/browse/DISC-1069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
